bindings/python/python/opendal/__init__.py

```diff
@@ -17,5 +17,4 @@
 
 from ._opendal import *
 
-__doc__ = _opendal.__doc__
```

Comment:
We don't need this anymore?

Reply:
This is just for the sake of documentation hierarchy. I believe the installation guide and quick start should appear on the front page of the documentation, rather than under the API reference.